partner_brand/models/partner.py

```diff
@@ -0,0 +1,15 @@
+# © 2023 David BEAL @ Akretion
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from odoo import fields, models
+
+
+class ResPartner(models.Model):
+    _inherit = "res.partner"
+
+    brand_id = fields.Many2one(comodel_name="res.brand")
+    brand_logo = fields.Image(
+        comodel_name="res.partner",
```

Comment:
`fields.Image` is not a related fields so we don't need this statement.

Reply:
Oops